# Microbiome Resources

List of software packages (and the people developing these methods) for microbiome (16S), metagenomics (WGS, Shot-gun sequencing).
Inspired by awesome-microbiome project (https://github.com/stevetsa/awesome-microbes/blob/master/README.md#awesome-microbes)
<br /> 


## Microbiome (16S)

[Qiime](http://qiime.org/) - QIIME is designed to take users from raw sequencing data generated on the Illumina or other platforms through publication quality graphics and statistics. This includes demultiplexing and quality filtering, OTU picking, taxonomic assignment, and phylogenetic reconstruction, and diversity analyses and visualizations.

[QIIME2](https://qiime2.org/) - QIIME2 is a new GUI for QIIME pakckage. It includes multiple plugins (DADA2 and Deblur etc). 

[mothur](https://www.mothur.org/)  - OTU-based analysis of 16S data.

[DADA2](http://benjjneb.github.io/dada2/) -  New sequence-based OTU identifications. 

[phyloseq](https://joey711.github.io/phyloseq/) - The R packages to process microbiome data sets.

[UPARSE](http://drive5.com/uparse/) - generates OTUs that are far superior to state-of-the-art methods including QIIME, mothur and AmpliconNoise on mock community tests. OTU representative sequences are more accurate predictions of biological sequences, and the number of OTUs is much close to the number of species.

[Qiita]( http://qiita.microbio.me/) - microbiome storage and analysis resource that can run on everything from your laptop to a supercomputer. It is built on top of the widely used QIIME package, and enables the exploration of -omics data.

[Nephele](https://nephele.niaid.nih.gov/#cloud)  - The web tools to analyze microbiome data on the cloud.

[PICRUST](http://picrust.github.io/picrust/) - The tools for pathway analysis using 16S data sets. 


## Metagenomics (WGS, Shot-gun sequencing)

[bioBakery-MetaPhlAn](https://bitbucket.org/biobakery/biobakery/wiki/Home) - a virtual environment platform that provides meta'omic analysis tools.

[Mauve](http://darlinglab.org/mauve/mauve.html) - a system for constructing multiple genome alignments in the presence of large-scale evolutionary events such as rearrangement and inversion. Multiple genome alignments provide a basis for research into comparative genomics and the study of genome-wide evolutionary dynamics.


## Public Data Sets

[HMP dataset](http://hmpdacc.org/HMQCP/): HMP data is microbiome data sets from around 250 healthy participants for 18 body sites. [Nature 2012 paper reference](https://www.nature.com/nature/journal/v486/n7402/full/nature11234.html)

[American Gut dataset](https://github.com/biocore/American-Gut/tree/master/data): American Gut project is microbiome data sets from more than 5000 individuals generated from Dr Rob Knight's group. (It is still updating)

## Data Format

[BIOM Format](http://biom-format.org/) - is designed to be a general-use format for representing biological sample by observation contingency tables. BIOM is a recognized standard for the Earth Microbiome Project and is a Genomics Standards Consortium supported project.





